src/map/skill.cpp

```diff
@@ -5069,7 +5069,7 @@ int skill_castend_damage_id (struct block_list* src, struct block_list *bl, uint
 	map_session_data *sd = NULL;
 	struct status_data *tstatus;
 	status_change *sc, *tsc;
-
+	struct homun_data *hd;
```

Comment:
= NULL is needed?

Reply:
As long as you don't check before the assignment, then no.